services/frontend/src/components/errorResult/errorPage.jsx

```diff
@@ -5,9 +5,8 @@ import { FormattedMessage } from "react-intl";
 import { HomeOutlined } from "@ant-design/icons";
 import ErrorResultView from "./errorResultView";
 
-const Error403 = () => {
+const ErrorPage = (error) => {
```

Comment:
Maybe rename the error to errorNumber? Also you are using the whole props here and I don't think that is what you meant to do. You will also need to add some propTypes to this component

```suggestion
const ErrorPage = ({ error }) => {
```